eng/versioning/version_client.txt

```diff
@@ -63,6 +63,8 @@ com.microsoft.azure:azure-spring-boot-test-core;2.2.4;2.2.5-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core;1.6.0-beta.1
+unreleased_com.azure:azure-core-http-netty;1.6.0-beta.1
```

Comment:
unreleased_com.azure:azure-core-http-netty has been added but isn't used anywhere

Reply:
Thanks! This is fixed now being used in azure-identity's pom.